src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+       href="#logo"
+       class="logo"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Here, `logo` seems to be more suitable for a logo, not for a link. Consider changing it to `logo__link` or similar.